doc/personas.md

```diff
@@ -0,0 +1,67 @@
+Personas
+========
```

Comment:
(non blocking) Just a personal favor but I'd prefer the markdown style with fewer characters:
```suggestion
# Personas
```

Reply:
done